src/Tizen.NUI/src/internal/Application.cs

```diff
@@ -1391,7 +1391,7 @@ internal static List<Window> GetWindowList()
             List<Window> WindowList = new List<Window>();
             for( uint i = 0; i < ListSize; ++i )
             {
-                Window currWin = new Window(Interop.Application.Application_GetWindowsFromList(i), true);
+                Window currWin = Registry.GetManagedBaseHandleFromNativePtr(Interop.Application.Application_GetWindowsFromList(i)) as Window;
                 if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
                 WindowList.Add(currWin);
             }
```

Comment:
when WindowList is created in line 1391, the ListSize will be Zero.
then after new currWin is added, the WindowList.Size will be increased.
ListSize is different from WindowList.Size ?
could you explain?